io/src/lib.rs

```diff
@@ -296,18 +289,6 @@ impl Write for Sink {
     fn flush(&mut self) -> Result<()> { Ok(()) }
 }
 
-#[cfg(feature = "std")]
-impl std::io::Write for Sink {
-    #[inline]
-    fn write(&mut self, buf: &[u8]) -> std::io::Result<usize> { Ok(buf.len()) }
-
-    #[inline]
-    fn write_all(&mut self, _: &[u8]) -> std::io::Result<()> { Ok(()) }
-
-    #[inline]
-    fn flush(&mut self) -> std::io::Result<()> { Ok(()) }
-}
```

Comment:
We can and should impl it for `BufWriter`.

Reply:
I'm at the point that I think we should just release this with whatever works for `rust-bitcoin` as it is. I don't know what is the plan for this crate in the future so I don't want to add anything that is not used. I just want to use the crate already in `hex` so I can fix stuff in `rust-bitcoin` that needs it.

Reply:
I don't think we should release total crap just because "it works" in our library. This is literally a few lines that will help with super common reasonable cases downstream. Without these actual IO would be insanely annoying to use.